pkg/services/tree/service.go

```diff
@@ -22,6 +22,9 @@ type Service struct {
 	closeCh     chan struct{}
 }
 
+// MaxGetSubTreeDepth represents maximum allowed traversal depth in GetSubTree RPC.
+const MaxGetSubTreeDepth = 10
```

Comment:
Don't forget to reflect this in protocol docs.

Reply:
Fixed.